common/app/templates/inlineJS/blocking/applyAdfreeRenderCondition.scala.js

```diff
@@ -0,0 +1,119 @@
+@()
```

Comment:
We've stopped using the single-`var` style, because it's a little unwieldy to edit. But that's a minor stylistic point.